rfcs/common-data.md

```diff
@@ -0,0 +1,159 @@
+---
+RFC: XXXX
+Author: Egor Baranov
+Status: Draft
+Created: 2017-05-03
+Last Modified: 2017-05-03
+---
+
+# API Blueprint RFC XXXX: Common Data
+
+## Table of Contents
+
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Rationale](#rationale)
+- [Backwards Compatibility](#backwards-compatibility)
+
+## Abstract
+
+This RFC proposes an ability to add common responses and to multiple actions.
+
+## Motivation
+
+When you describe big API several actions can have identical responses.
+For example you have 10 actions allowed for authorized user only, so each
+of such actions can respond with 401. That is why you have to write same code
+10 times. If you want to change this code you have to change it in 10 places.
+This problem can be cause of misspellings and logical errors.
+So ability to add common responses in one place for multiple actions is required.
```

Comment:
The authentication framework is a great idea, but I think that authentication is only one problem which can be solved using `Common Data`. Another example: we use some parter's API for several actions. Sometimes it does not work, so we return `externalAPIError`. It is common response but it is not connected with authentication.